docs/pages/cheatsheet.adoc

```diff
@@ -432,6 +485,15 @@ Removed from service object.
 |text           | string expression     | The textual representation of the link.
 |====================
 
+*Annotations:*
+
+[options="header", cols="1,2"]
+|====================
+|Annotation | Meaning
+|@Lazy      | Defer fetching the object(s) as long as it is possible.
```

Comment:
If we want to mimic how the current frontend handles these relations, I can help write the description. If this is something different, then we need a more concrete description on how this supposed to work.

Reply:
rephrased